app/assets/javascripts/discourse/widgets/post-cooked.js

```diff
@@ -273,19 +292,29 @@ export default class PostCooked {
     });
   }
 
+  _createDetachedElement(nodeName) {
```

Comment:
Minor but this method doesn't access any instance variables and could be more private by putting it at the top of the module as a regular function.

Reply:
:+1: done, thanks. I also refactored so that the detached document is re-used, rather than recreating each time